addons/addon-base-raas/packages/base-raas-services/lib/environment/environment-mount-service.js

```diff
@@ -230,7 +242,7 @@ class EnvironmentMountService extends Service {
           s3Policy.Statement = s3Policy.Statement.filter(
             statement => ![listSid, getSid, putSid].includes(statement.Sid),
           );
-          [listStatement, getStatement, putStatement].forEach(statement => {
+          [listStatement, getStatement, putStatement, secureTransportStatement].forEach(statement => {
```

Comment:
Add a unit test for this new statement?

Reply:
created a helper method to generate these statements and added a unit test for that method per Jeet's advice